assets/js/script.js

```diff
@@ -44,6 +48,9 @@ function init() {
 }
 
 function petFinderCall() {
+    //Clears array each time the Submit button is clicked by user so that we aren't getting previous searches
+    arrayOfPetsInQueue = [];
```

Comment:
should not clear everytime it calls because adding any would skip 10 gets cleared when the submit button gets clicked